themes/finna2/templates/RecordDriver/DefaultRecord/data-termsOfUse.phtml

```diff
@@ -2,10 +2,11 @@
 <?php foreach ($data as $field): ?>
   <?=!empty($field['material']) ? $this->escapeHtml($field['material']) . ': ' : '' ?>
   <?=!empty($field['terms']) ? $this->escapeHtml($field['terms']) : '' ?>
+  <?=!empty($field['sourcecode']) ? $this->escapeHtml($this->translate('accessrestriction_source_' . $field['sourcecode'])) : '' ?>
```

Comment:
Käytä `$this->transEscWithPrefix('accessrestriction_source_', $field['sourcecode'])`. Se näyttää sitten pelkän sourcecode:n, jos ei ole käännöstä.